### PR TITLE
WIP Sprint 24 version check

### PR DESCRIPTION
**Pull request by jh-bate**
**DO NOT MERGE**

@jebeck here are the changes I was looking at. As discussed they will change with your changes to how the uploader is structured.

### CODE REVIEW

File: lib/state/appActions.js [hunk 1]
```diff
@@ -247,6 +250,12 @@ appActions.load = function(cb) {
 
 appActions.login = function(credentials, options, cb) {
   var self = this;
+
+  var err = appActions._assertUploaderVersion();
```
Comment (jh-bate) [inline lib/state/appActions.js]: so I am trying to assert that the version of the uploader they are using is allowed to upload to the platform. If it isn't a required version then they are prompted to update before going any further

File: lib/state/appActions.js [hunk 2]
```diff
@@ -579,6 +588,11 @@ appActions.upload = function(uploadIndex, options, cb) {
   var self = this;
   cb = cb || _.noop;
 
+  var err = appActions._assertUploaderVersion();
```
Comment (jh-bate) [inline lib/state/appActions.js]: Might be un-necessary as hopefully they have been prompted to update before if there uploader is out of date

File: lib/state/appActions.js [hunk 3]
```diff
@@ -627,6 +643,18 @@ appActions.upload = function(uploadIndex, options, cb) {
     appActions.addMoreInfoToError(error, appActions.errorStages['STAGE_UPLOAD']);
     throw error;
   }
+
+
+
+};
+
+appActions._assertUploaderVersion = function() {
```
Comment (jh-bate) [inline lib/state/appActions.js]: the basic assert that returns an error if they need to update their uploader

**Comment by jebeck** [comment]
This will be included in #219 so closing for clarity.

**Pull request closed without merging**